xds/internal/client/cds_test.go

```diff
@@ -139,24 +141,38 @@ func (s) TestValidateCluster_Success(t *testing.T) {
 		{
 			name: "happy-case-logical-dns",
 			cluster: &v3clusterpb.Cluster{
+				Name:                 clusterName,
 				ClusterDiscoveryType: &v3clusterpb.Cluster_Type{Type: v3clusterpb.Cluster_LOGICAL_DNS},
 				LbPolicy:             v3clusterpb.Cluster_ROUND_ROBIN,
 			},
-			wantUpdate: ClusterUpdate{ServiceName: "", EnableLRS: false, ClusterType: ClusterTypeLogicalDNS},
+			wantUpdate: ClusterUpdate{ClusterName: clusterName, EnableLRS: false, ClusterType: ClusterTypeLogicalDNS},
 		},
 		{
 			name: "happy-case-aggregate-v3",
 			cluster: &v3clusterpb.Cluster{
+				Name: clusterName,
 				ClusterDiscoveryType: &v3clusterpb.Cluster_ClusterType{
-					ClusterType: &v3clusterpb.Cluster_CustomClusterType{Name: "envoy.clusters.aggregate"},
+					ClusterType: &v3clusterpb.Cluster_CustomClusterType{
+						Name: "envoy.clusters.aggregate",
+						TypedConfig: func() *anypb2.Any {
```

Comment:
We now have a utility function to do this:
https://github.com/grpc/grpc-go/blob/master/internal/testutils/marshal_any.go#L30

Reply:
Done